closed/JPP.gmk

```diff
@@ -38,6 +38,10 @@ ifeq (true,$(BUILD_OPENJCEPLUS))
   JPP_TAGS += OPENJCEPLUS_SUPPORT
 endif # BUILD_OPENJCEPLUS
 
+ifeq (true,$(OPENJ9_ENABLE_JFR))
+  JPP_TAGS += JFR_SUPPORT
+endif # OPENJ9_ENABLE_JFR
+
```

Comment:
This should be added before line 37 like newer versions. A change should be proposed and merged for jdk17 before this is considered.

Reply:
ok, Opened https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/390

Reply:
Please add for JDK21, JDK23 and JDKNext as well

Reply:
The `JFR_SUPPORT` tag is already supported in jdk21 and newer versions.